posthog/urls.py

```diff
@@ -90,6 +90,8 @@ def handler500(request):
 
 @ensure_csrf_cookie
 def home(request, *args, **kwargs):
+    if settings.REDIRECT_APP_TO_US and request.build_absolute_uri().startswith("http://app.posthog.com"):
```

Comment:
should this be https?

```suggestion
    if settings.REDIRECT_APP_TO_US and request.build_absolute_uri().startswith("https://app.posthog.com"):
```